code/Model/EditableFormField.php

```diff
@@ -772,7 +772,7 @@ protected function updateFormField($field)
         }
 
         // if this field has a placeholder
-        if ($this->Placeholder || $this->Placeholder === '0') {
+        if (strlen($this->Placeholder >= 0)) {
```

Comment:
Got some bracket placement issues here 🙂

Reply:
I see that @dhensby had the same bracket ordering but I think that must've been a typo. `strlen($this->Placeholder >= 0)` makes very little sense.

Reply:
Bracket has been moved.